launcher/src/main/java/org/wso2/carbon/launcher/config/CarbonLaunchConfig.java

```diff
@@ -58,6 +59,8 @@
 public class CarbonLaunchConfig {
 
     private static final Logger logger = Logger.getLogger(CarbonLaunchConfig.class.getName());
+    private static final String VAR_REGEXP = "(?<=@)(\\d)";
+    private static final Pattern varPattern = Pattern.compile(VAR_REGEXP);
```

Comment:
Let's use a consistent naming convention for static final variables here

Reply:
Fixed in commit [#d29217fc87e92c320e1e83dc9a03374cdc6a23ff](https://github.com/wso2/carbon-kernel/pull/1775/commits/d29217fc87e92c320e1e83dc9a03374cdc6a23ff)